src/include/com/amazonaws/kinesis/video/webrtcclient/Stats.h

```diff
@@ -593,20 +607,17 @@ typedef struct {
                                      //!< In all of these cases the error callback (if specified) will be called.
     UINT32 numberOfReconnects;       //!< Number of reconnects in the session
     UINT32 apiCallRetryCount;        //!< Number of retries due to API call failures in the state machine
-    UINT64 getTokenCallTime;         //!< Time (ms) taken to get credentials for signaling
```

Comment:
You are removing members of a public struct which will break backward compatibility. Unless there is a strong reason to make this change, the existing implementations should not change. If the goal is to record start and end time, let us record them in the new members and not change any existing calculations. Same applies to wherever we are removing public members.

Reply:
+1.  We should not remove these.